### DISCUSSION TITLE
Offline install on CentOS 7

### DISCUSSION THREAD

**Discussion by jadick4** [Security-Onion-Solutions/securityonion#1721] [Unsupported Versions] [answered]
Has anyone cloned and locally hosted the resources necessary for installing with the so-setup-network script after cloning with git?  Something similar to cloning a yum repository and running a small web server and hosting it for a offline network.  Due to the needed configuration it has to be installed air-gaped and the .ISO does not allow for RAID or LVM of multiple disks during install unless I am missing something.

**Comment by TOoSmOotH** [comment, marked as answer]
You can always add more disks to the LVM post install and extend the file systems. Raid on the other hand is a little more difficult if using software raid.
